### PR TITLE
Added Power Operator using ** and nullish operator ??

### PR DESCRIPTION
**Pull request by manan657**

### CODE REVIEW

**Comment by ankitchouhan1020** [comment]
Please add relevant test cases to support these features.

File: packages/interpreter/src/helpers/index.ts [hunk 1]
```diff
@@ -132,6 +132,15 @@ export function getOperationValue(
     case "||":
       return operands.left || operands.right;
 
+    case '**':
+      if (checkNumberOperands(operands)) {
+        return Math.pow(operands.left,operands.right);
+      }
+      throw exception;
```
Comment (ghost) [inline packages/interpreter/src/helpers/index.ts:139]: How about throwing with meaningful message, unless that is part of exception already.

Comment (Deep-Codes) [reply packages/interpreter/src/helpers/index.ts:139]: it already throws a meaningful message. try it out in the playground.

Comment (manan657) [reply packages/interpreter/src/helpers/index.ts:139]: image

Meaningful message is already there.

**Comment by manan657** [comment]
> Please add relevant test cases to support these features.



> Collaborator

I have added the test cases.

File: packages/interpreter/src/helpers/index.ts (hunk 1 above)
Comment (manangupta657) [reply packages/interpreter/src/helpers/index.ts:139]: Done

**Comment by manangupta657** [comment]
> Please add relevant test cases to support these features.

Done

**Review by manangupta657 — APPROVED** [review]

**Comment by aniketsingh0104** [comment]
You haven't implemented this in parser, it won't even compile. Please take a look at parser first. Closing this.

**Pull request closed without merging**